nutkit/protocol/requests.py

```diff
@@ -592,3 +592,19 @@ def __init__(self, result_id, record_key, type_name, field_id):
         self.recordKey = record_key
         self.type = type_name
         self.field = field_id
+
+
+class ExecuteQuery:
+    """
+    Request to execute a query in a retriable context.
+
+    Backend should return EagerResult or a Error response.
```

Comment:
Is it worth documenting the fields, their types, and meaning? Especially the `config.bookmarkManagerId` values that have a special meaning.

Reply:
I've put some docs in a semi structured way.